tests/e2e/lib.sh

```diff
@@ -43,42 +43,48 @@ deploy_stackrox() {
 }
 
 # shellcheck disable=SC2120
-deploy_stackrox_with_custom_sensor() {
-    if [[ "$#" -ne 1 ]]; then
-        die "expected sensor chart version as parameter in deploy_stackrox_with_custom_sensor"
+deploy_stackrox_with_custom_central_and_sensor_versions() {
+    if [[ "$#" -ne 2 ]]; then
+        die "expected central chart version and sensor chart version as parameters in deploy_stackrox_with_custom_central_and_sensor_versions: deploy_stackrox_with_custom_central_and_sensor_versions <central chart version> <sensor chart version>"
     fi
-    target_version="$1"
-    setup_podsecuritypolicies_config
-
-    deploy_central
-
-    export_central_basic_auth_creds
-    wait_for_api
-    setup_client_TLS_certs "${2:-}"
-    record_build_info
+    ci_export CENTRAL_CHART_VERSION_OVERRIDE "$1"
+    ci_export SENSOR_CHART_VERSION_OVERRIDE="$2"
+    ci_export DEPLOY_STACKROX_VIA_OPERATOR "false"
+    ci_export OUTPUT_FORMAT "helm"
+    ci_export DISABLE_RHACS_IMAGE_REPOSITORY_PARAMS "true"
+    unset COLLECTOR_IMAGE_REPO
 
-    # generate init bundle
-    password_file="$ROOT/deploy/$ORCHESTRATOR_FLAVOR/central-deploy/password"
-    if [ ! -f "$password_file" ]; then
-        die "password file $password_file not found after deploying central"
-    fi
-    kubectl -n stackrox exec deploy/central -- roxctl --insecure-skip-tls-verify \
-        --password "$(cat "$password_file")" \
-      central init-bundles generate stackrox-init-bundle --output - 1> stackrox-init-bundle.yaml
+    helm repo add stackrox-oss https://raw.githubusercontent.com/stackrox/helm-charts/main/opensource
+    helm repo update
 
-    deploy_sensor_from_helm_charts "$target_version" ./stackrox-init-bundle.yaml
+    latest_tag=$(make tag)
 
-    echo "Sensor deployed. Waiting for sensor to be up"
-    sensor_wait
+    helm_charts=$(helm search repo stackrox-oss -l)
+    central_regex="stackrox-oss/stackrox-central-services[ \t]*.${CENTRAL_CHART_VERSION_OVERRIDE}[ \t]*.([0-9]+\.[0-9]+\.[0-9]+)"
+    sensor_regex="stackrox-oss/stackrox-secured-cluster-services[ \t]*.${SENSOR_CHART_VERSION_OVERRIDE}[ \t]*.([0-9]+\.[0-9]+\.[0-9]+)"
 
-    # Bounce collectors to avoid restarts on initial module pull
-    kubectl -n stackrox delete pod -l app=collector --grace-period=0
+    if  [[ $helm_charts =~ $central_regex ]]; then
+        ci_export CENTRAL_CHART_DIR_OVERRIDE "stackrox-oss/stackrox-central-services"
+    elif [[ "$latest_tag" != "$CENTRAL_CHART_VERSION_OVERRIDE" ]]; then
+        echo >&2 "stackrox-central-services helm chart for version ${CENTRAL_CHART_VERSION_OVERRIDE} not found in stackrox-oss repo nor is it the latest tag."
+        exit 1
+    fi
 
-    sensor_wait
+    if [[ $helm_charts =~ $sensor_regex ]]; then
+        sensor_chart_dir_override="stackrox-oss/stackrox-secured-cluster-services"
+    elif [[ "$latest_tag" == "$SENSOR_CHART_VERSION_OVERRIDE" ]]; then
+        sensor_chart_dir_override="latest-secured-cluster-services"
```

Comment:
Could you please make this look like a path so that it's more clear when looking at the command on the output, e.g. currently it's unclear whether `latest-secured-cluster-services` is a reference or a directory
```
helm upgrade --install -n stackrox --create-namespace stackrox-secured-cluster-services latest-secured-cluster-services -f /go/src/github.com/stackrox/stackrox/deploy/k8s/sensor-deploy/init-bundle.yaml --set imagePullSecrets.allowNone=true --set clusterName=remote --set centralEndpoint=central.stackrox:443 --set collector.collectionMethod=EBPF --set env.openshift=false --set helmManaged=false --set customize.envVars.ROX_RESYNC_DISABLED=true --set admissionControl.listenOnCreates=true --set admissionControl.listenOnUpdates=true --set admissionControl.listenOnEvents=true
```

```suggestion
        sensor_chart_dir_override="./latest-secured-cluster-services"
```

Reply:
Ok